File: src/utils/merge_csvs.scala
from pyspark.sql import SparkSession
import pyspark.sql.functions as func


var df = spark.read.format("csv").\
         option("header", "true").\
         option("inferSchema", "true").\
         load("./data/*.csv")

df.write.csv("boston_2015_to_2021.csv")

val parsed = spark.read.
 option("header", "true").
 option("nullValue", "?").
 option("inferSchema", "true").
 csv("./data/*.csv")

File: src/utils/merge_csvs.py
from pyspark.sql import SQLContext
from pyspark.sql.types import *

sqlContext = SQLContext(sc)

var df = spark.read.format("csv").option("header", "true").option("inferSchema", "true").load("./data/*.csv")

df.write.csv("boston_2015_to_2021.csv")
